doc/source/docs/howto_build_docs.rst

```diff
@@ -4,76 +4,100 @@
 
-We currently use Sphinx_ for generating the API and reference
-documentation for NumPy.  You will need Sphinx >= 2.2.0.
-
 If you only want to get the documentation, note that pre-built
 versions can be found at
 
     https://numpy.org/doc/
 
 in several different formats.
 
-.. _Sphinx: http://www.sphinx-doc.org/
 
+Prerequisites
+-------------
 
-Instructions
-------------
+Before proceeding further it should be noted that the documentation is built with the ``make`` tool,
+which is not available on Windows. It is recommended for Windows users to set up their development
+environment on :ref:`Gitpod <development-gitpod>` or `Windows Subsystem
+for Linux (WSL) <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`_.
+Gitpod is recommended if you have good internet connectivity and want a temporary setup.
+WSL is a good option for a persistent local setup.
+
+Building the NumPy documentation and API reference requires the following:
+
+1) NumPy
+
+   Since large parts of the main documentation are obtained from NumPy via
+   ``import numpy`` and examining the docstrings, you will need to first
+   :ref:`build <building-from-source>`, and install it so that the correct version is imported.
+   NumPy has to be re-built and re-installed every time you fetch the latest version of the
+   repository, before generating the documentation. This ensures that the NumPy version and
+   the git repository version are in sync.
 
-If you obtained NumPy via git, get also the git submodules that contain
-additional parts required for building the documentation::
+   Note that you can eg. install NumPy to a temporary location and set
+   the PYTHONPATH environment variable appropriately.
+   Alternatively, if using Python virtual environments (via e.g. ``conda``,
+   ``virtualenv`` or the ``venv`` module), installing NumPy into a
+   new virtual environment is recommended.
 
-    git submodule update --init
+2) Dependencies
 
-In addition, building the documentation requires the Sphinx extension
-`plot_directive`, which is shipped with Matplotlib_. This Sphinx extension can
-be installed by installing Matplotlib. You will also need Python>=3.6.
+   All of the necessary dependencies for building the NumPy docs can be installed
+   with::
 
-Since large parts of the main documentation are obtained from numpy via
-``import numpy`` and examining the docstrings, you will need to first build
-NumPy, and install it so that the correct version is imported.
+       pip install -r doc_requirements.txt
 
-After NumPy is installed, install SciPy since some of the plots in the random
-module require `scipy.special` to display properly.
+   We currently use Sphinx_ (>= 2.2.0) for generating the API and reference
+   documentation for NumPy. In addition, building the documentation requires
+   the Sphinx extension `plot_directive`, which is shipped with Matplotlib_. SciPy_
+   is installed since some of the plots in the random module require `scipy.special`
```

Comment:
There are other places - for example in the SVD tutorial SciPy is also needed. Maybe just say "some examples and documents require the use of SciPy functions".